public/css/master.css

```diff
@@ -36,3 +36,39 @@ nav li {
   margin: auto;
   max-width: 600px;
 }
+/*placeholder styles are temp*/
+.placeholder-video {
+  border: solid black .5em;
+  background: silver;
+}
+.main-video {
+  height: 20em;
+  width: 36em;
+  max-width: 960px;
+  margin: auto;
+  margin-top: 5em;
+}
+.video-caption {
+  text-align: center;
+  padding-top: 1em;
+}
+.videos-container {
+  display: flex;
+  flex-direction: row;
```

Comment:
Do you know what your target devices are?

From caniuse.com
http://caniuse.com/#search=flexbox

![image](https://user-images.githubusercontent.com/12845233/28568995-caf0223e-712f-11e7-8fe5-e0113bbc8800.png)

I you are not adding all the prefixes then you only have support on 84/87% of browsers..

If you add prefixes you have 84/97%.

https://css-tricks.com/forums/topic/flexbox-prefixes/